jkind/src/jkind/analysis/StaticAnalyzer.java

```diff
@@ -25,13 +27,13 @@
 import jkind.util.Util;
 
 public class StaticAnalyzer {
-	public static void check(Program program, SolverOption solver) {
-		checkErrors(program, solver);
+	public static void check(Program program, SolverOption solver, Settings settings) {
```

Comment:
Provide a version that doesn't take the `settings` for people who don't care about them.